atd-vzv/src/views/summary/CrashesByTimeOfDay.js

```diff
@@ -182,17 +202,43 @@ const CrashesByTimeOfDay = () => {
             series={
               <HeatmapSeries
                 colorScheme={[
-                  colors.intensity1Of5Lowest,
                   colors.intensity2Of5,
                   colors.intensity3Of5,
                   colors.intensity4Of5,
                   colors.viridis1Of6Highest,
                 ]}
+                emptyColor={colors.intensity1Of5Lowest}
+                cell={
+                  <HeatmapCell
+                    tooltip={
+                      <ChartTooltip
+                        content={(d) =>
+                          `${d.x} ∙
+                          ${formatValue(d)}`
+                        }
+                      />
+                    }
+                  />
+                }
               />
             }
           />
         </Col>
       </Row>
+      <Row>
+        <Col className="pb-2">
```

Comment:
Could we add some padding above the legend? maybe just update `pb-2` to `py-2`.

Reply:
Done!